packages/node/src/node.ts

```diff
@@ -42,6 +69,21 @@ export default class Node {
     this.incoming.emit(event, req);
   }
 
+  get address() {
+    return this.signer.address;
+  }
+
+  /**
+   * Sends a message to another Node. It also auto-includes the from field
+   * in the message.
+   * @param peerAddress The peer to whom the message is being sent.
+   * @param msg The message that is being sent.
+   */
+  async send(peerAddress: Address, msg: object) {
+    const modifiedMsg = Object.assign({ from: this.address }, msg);
```

Comment:
We don't care about the browser compatibility because we're compiling this code and using rollup which will probably just use `Object.assign` anyway. So it's purely a code cleanliness this from my point of view.

Reply:
The syntactic difference is pretty negligible here IMO, but I'm mostly indifferent.